README.md

```diff
@@ -16,97 +22,70 @@
-### Delete the default app installed by the operator
-```bash
-oc delete argocd openshift-gitops -n openshift-gitops
-```
-### Deploy a argocd app with vault plugin
-```bash
-oc apply -f openshift-gitops/infra/argocd.yaml
+$ ./scripts/infra/argocd.sh provision
+subscription.operators.coreos.com/openshift-gitops-operator created
+sleep 10 seconds until argocd subscription is installed.
+Operator subscription installed successfully
+Deploy ArgoCD app
+serviceaccount/argocd-repo-server created
+argocd.argoproj.io/redhat-kong-gitops created
+sleep 10 seconds until argocd app is Available.
+ArgoCD app deployed successfully
 ```
 
-### Get the route and password of argocd gui
+#### Get the route and password of argocd gui
+
 ```bash
 oc get routes -n openshift-gitops redhat-kong-gitops-server --template='{{ .spec.host }}'
 ```
+
 ```bash
 oc get secret -n openshift-gitops redhat-kong-gitops-cluster -ojsonpath='{.data.admin\.password}' | base64 -d
 ```
 
-### Add controlplane and dataplane cluster
+#### Add controlplane and dataplane clusters
+
 ```bash
-export ARGOCD_SERVER_URL=$(oc get routes -n openshift-gitops | grep redhat-kong-gitops-server | awk '{print $2}')
 oc get secret -n openshift-gitops redhat-kong-gitops-cluster -ojsonpath='{.data.admin\.password}' | base64 -d
-argocd login $ARGOCD_SERVER_URL
-argocd cluster add dp
-argocd cluster add cp
+argocd login `oc get routes -n openshift-gitops redhat-kong-gitops-server --template='{{.spec.host}}'`
+argocd cluster add -y --name dp <dp-context>
+argocd cluster add -y --name cp <cp-context>
 ```
 
-### Create the project for control plane and data plane
-The deployment of kong gateway will be on two unmanaged clusters define the [following fields](./openshift-gitops/infra/project.yaml)
-```yaml
-apiVersion: argoproj.io/v1alpha1
-kind: AppProject
-metadata:
-  name: dataplane
-  namespace: openshift-gitops
-  annotations:
-    apps: <<todo>>
-spec:
-  destinations:
-  - namespace: '*'
-    server: <<todo>>>
-  clusterResourceWhitelist:
-  - group: '*'
-    kind: '*'
-  sourceRepos:
-  - <<todo>>
----
-apiVersion: argoproj.io/v1alpha1
-kind: AppProject
-metadata:
-  name: controlplane
-  namespace: openshift-gitops
-spec:
-  destinations:
-  - namespace: '*'
-    server: <<todo>>
-  clusterResourceWhitelist:
-  - group: '*'
-    kind: '*'
-  sourceRepos:
-  - <<todo>>
-```
+#### Create the project for control plane and data plane
 
-- [TODO] - Ruben - Automate the above?
+##### Use defaults
 
 ```bash
+kustomize build openshift-gitops/infra/base | kubectl apply -f -
+```
 
-oc apply -f openshift-gitops/infra/project.yaml
+##### Use a custom repository / branch
+
+Copy the `repositories.yaml.template` file in [./openshift-gitops/infra/overlays](./openshift-gitops/infra/overlays) to `repositories.yaml` and edit the values if you need
+to update the repository path or branch where the charts will be installed from.
+
+```bash
+kustomize build openshift-gitops/infra/overlays | kubectl apply -f -
 ```
```

Comment:
Before you fixed the namespace, it was working, now it no longer works

Reply:
`kustomize build openshift-gitops/infra/overlays` gives this error. Error `Error: no matches for Id AppProject.v1alpha1.argoproj.io/dataplane.[noNs]; failed to find unique target for patch AppProject.v1alpha1.argoproj.io/dataplane.[noNs]`. When you fix this, I can test and merge

Reply:
make sure you updated your `repositories.yaml` file with the latest `repositories.yaml.template` file